api/app/fireweather_bot/noon_forecasts.py

```diff
@@ -46,6 +46,8 @@ def _parse_csv(temp_path: str):
     with open(temp_path, 'r') as csv_file:
         data_df = pd.read_csv(csv_file)
     station_codes = get_station_names_to_codes()
+    # drop any rows where 'display_name' is not found in the station_codes lookup:
+    data_df.drop(index=data_df[~data_df['display_name'].isin(station_codes.keys())].index, inplace=True)
```

Comment:
isn't this dropping all rows where the display_name is **in** station_codes ?

Reply:
I am guessing `~` is a not operator in this context?

Reply:
doh! yes, `~` is not. completely missed that

Reply:
yeah - took me a while to figure out how to do a "is not in" on a dataframe.